alert/log/rawentry.go

```diff
@@ -5,11 +5,12 @@ import (
 	"database/sql"
```

Comment:
Worth renaming the file from `rawentry.go` -> `entry.go`?

Reply:
yes, missed that